cmake/CMakeLists.txt

```diff
@@ -461,7 +465,7 @@ file(GLOB IFCGEOM_CPP_FILES ../src/ifcgeom/*.cpp)
 set(IFCGEOM_FILES ${IFCGEOM_CPP_FILES} ${IFCGEOM_H_FILES})
 IF(BUILD_SHARED_LIBS)
 	message(WARNING "Building IfcGeom as shared library not currently supported")
-        add_library(IfcGeom STATIC ${IFCGEOM_FILES})
+	add_library(IfcGeom SHARED ${IFCGEOM_FILES})
```

Comment:
It would be sufficient to move `add_library()` outside of the if statement. `BUILD_SHARED_LIBS=1` sets `add_library` default to `STATIC`.